tests/views/test_collection_exercise.py

```diff
@@ -1277,26 +1276,6 @@ def test_create_collection_exercise_success(self, mock_request):
         self.assertEqual(response.status_code, 200)
         self.assertIn("123456".encode(), response.data)
 
-    @requests_mock.mock()
-    def test_create_collection_exercise_failure(self, mock_request):
```

Comment:
:thumbsdown: This test is not good, and is testing bad practise in our code. We should not just let a 500 from another service bubble up through this service as well and produce a 500, which is what it is testing. The generic assert request_history should also not be used like this as multiply mock_requests are in play and therefore renders a generic history count pointless. The good news is this doesn't actually cover a single line of code above the happy path and can be removed.